src/main/java/de/tum/in/www1/artemis/domain/scores/StudentScore.java

```diff
@@ -0,0 +1,70 @@
+package de.tum.in.www1.artemis.domain.scores;
+
+import javax.persistence.*;
+
+import de.tum.in.www1.artemis.domain.DomainObject;
+import de.tum.in.www1.artemis.domain.Exercise;
+import de.tum.in.www1.artemis.domain.Result;
+import de.tum.in.www1.artemis.domain.User;
+
+@Entity
+@Table(name = "student_score")
+public class StudentScore extends DomainObject {
+
+    @ManyToOne
+    private User student;
+
+    @ManyToOne
+    private Exercise exercise;
+
+    /**
+     * A submission can have a result and therefore, results are persisted and removed with a submission.
```

Comment:
I do not understand this comment at this position?

Reply:
True. No idea. Removed.